docs/01-overview/main-areas/observability/obsv-10-useful-links.md

```diff
@@ -10,6 +10,7 @@ If you're interested in learning more about the Observability area, check out th
 
 - Install a [custom Loki stack](https://github.com/kyma-project/examples/tree/main/loki).
 - Install a [custom Jaeger stack](https://github.com/kyma-project/examples/tree/main/jaeger).
+- Install a [custom Prometheus stack](https://github.com/kyma-project/examples/tree/main/prometheus).
```

Comment:
Let's also link [this](https://github.com/kyma-project/examples/pull/236) once it's merged.

Reply:
okay, I will keep an eye on it and will link it once it's merged

Reply:
It's merged now.

Reply:
The link is added now